packages/graphback/src/resolvers/knex/resolverImplementation.ts

```diff
@@ -2,8 +2,9 @@ const knexContext = `context.db`
 const argumentContext = `args`
 
 export const createTemplate = (fieldName: string, tableName: string): string => {
-    return `${fieldName}: (root, args, context, info) => {
-      return ${knexContext}('${tableName}').insert(${argumentContext}.input).returning('*')
+    return `${fieldName}: async (root, args, context, info) => {
+      const result = await ${knexContext}('${tableName}').insert(${argumentContext}.input).returning('*')
```

Comment:
Formatting of those can be off. 
We can do:

A) Ignore formatting completely and use external formatter after (like vscode headless) based on the editorconfig in the template (we need to add it actually)
B) Try to format code ourselves here to look nicely without formatter with some opiniated format (spaces vs tabs ;P)

Ideally, we need both :)